nl/guides/exercises/creating-exercises/management/index.md

```diff
@@ -11,11 +11,10 @@ next: false
 Nu je een oefening hebt die je kan gebruiken op Dodona, zijn er tal van dingen die je nog kan doen.
 Hieronder enkele suggesties voor volgende stappen:
 
-- Los je oefening zelf eens op in Dodona.
-- Laat je oefening eens oplossen door iemand anders. Zo ontdek je vaak nog gaten in het testplan, en kan je de oefening verder verfijnen.
 - [Maak een cursus aan](/nl/guides/teachers/creating-a-course/).
-- [Gebruik je oefening in een cursus](/nl/guides/teachers/exercise-series-management/).
+- [Gebruik je oefening in een oefeningenreeks van een cursus](/nl/guides/teachers/exercise-series-management/).
 - Maak nog andere soorten oefeningen.
+- Maak [leesactiviteiten](/nl/references/exercise-config/) met extra informatie of kleine stukken theorie.
```

Comment:
Note (to self): in the future, we'll want to link to a guide, https://github.com/dodona-edu/dodona-edu.github.io/issues/359